helm/fiftyone-teams-app/README.md.gotmpl

```diff
@@ -319,6 +320,23 @@ FiftyOne Teams continues to support the use of environment variables to set
 storage credentials in the application context and is providing an alternate
 configuration path.
 
+### Static Banner Configuration
+
+Fiftyone Teams v2.5 introduces the ability to add a static banner to the
+application.
+
+Configure the Static Banner by setting the following environment variables in
+your `values.yaml`.
+
+Banner text is configured with:
+`casSettings.env.FIFTYONE_APP_BANNER_TEXT` and
+`teamsAppSettings.env.FIFTYONE_APP_BANNER_TEXT`
+
+
+Banner background color is configured with:
+`casSettings.env.FIFTYONE_APP_BANNER_COLOR` and
+`teamsAppSettings.env.FIFTYONE_APP_BANNER_COLOR`
```

Comment:
Should we provide some hint at what values should be set?  Hex number? String like "blue"? etc?

Reply:
technically anything css understands is possible - hex, rgc, string. cc @Br2850 in case there was some dicussion I missed

Reply:
While we may know this, a customer reading these docs doesn't.  We should guide them by providing more information and even examples.